wiremock-spring-boot/src/main/java/com/maciejwalkowiak/wiremock/spring/ConfigureWireMock.java

```diff
@@ -30,11 +30,11 @@
     String name();
 
     /**
-     * The name of Spring property to inject the {@link WireMockServer#baseUrl()}
+     * Names of Spring properties to inject the {@link WireMockServer#baseUrl()}.
      *
-     * @return the name of Spring property to inject the {@link WireMockServer#baseUrl()}
+     * @return names of Spring properties to inject the {@link WireMockServer#baseUrl()}.
      */
-    String property() default "";
+    String[] property() default "";
```

Comment:
current implementation is fully backward compatible. 

But its readability is not as good as it could be when renaming the attribute to `properties`, is it ?

Reply:
Indeed. Add `properties` and mark `property` as deprecated. For the time being we will support both, later the `property` will be removed.